Assets/Scripts/CardController.cs

```diff
@@ -4,15 +4,26 @@
 
 public class CardController : MonoBehaviour
 {
+
+    public HandController hand;
+
+    SpriteRenderer spriteRenderer;
+
+
+    public Color32 defaultColor;
```

Comment:
totally up2u but I would use selectedColor and unselectedColor, rather than default and highlighted.

Reply:
Agreed. Changed.